src/main/java/de/hysky/skyblocker/mixins/InGameHudMixin.java

```diff
@@ -159,4 +164,19 @@ public abstract class InGameHudMixin {
     private void skyblocker$afterDrawersInitialized(CallbackInfo ci) {
         this.layeredDrawer.addLayer(HudRenderEvents.LAST.invoker()::onRender);
     }
+
+    @Inject(method = "renderPlayerList", at = @At("HEAD"))
+    private void skyblocker$renderHud(DrawContext context, RenderTickCounter tickCounter, CallbackInfo ci) {
```

Comment:
since the `RenderTickCounter` instance isn't used, swap to using an `@Local(argsOnly = true) DrawContext context`

Reply:
uh? am confusled

Reply:
use the @Local to capture the DrawContext because the RenderTickCounter is not used

Reply:
Done, maybe vic should look at f01d2b6 if confused.